presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -116,6 +117,7 @@
 {
     private static final Logger log = Logger.get(SemiTransactionalHiveMetastore.class);
     private static final int PARTITION_COMMIT_BATCH_SIZE = 8;
+    private static final int MAX_LAST_DATA_COMMIT_TIME_ENTRY = 100_000;
```

Comment:
This is a huge number per table. Suppose we have 50 running queries. Each query touches 4 tables and each entry has 100B in size. That easily takes us 20GB memory. Can we make it 100 or less?

Reply:
Can we address this comment (https://github.com/prestodb/presto/pull/17830/files#r927416793)?

Reply:
Yeah, changed to 100. If DEPA team has problems with it, we can revisit it later.